modules/ROOT/pages/install-port-reqs.adoc

```diff
@@ -5,6 +5,13 @@ endif::[]
 
 Runtime Fabric requires specific network and port configuration for installation and operation.
 
+== Bandwidth Requirements
+
+A high bandwidth network is required, with a minimum of 1 GBps connectivity in the internal network running the RTF nodes. 
+[NOTE]
+Due to data center connectivity being lower than the minimum requirement, cross-regional deployments of a Runtime Fabric 
```

Comment:
We can be blunt here: "Cross-regional deployments of a single Runtime Fabric is not supported."
If we list reasons, customers may claim they have a set up that doesn't apply.

Reply:
I removed that part - thank you!